components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -89,22 +90,23 @@ finish_setup() {
   for pkg in $pkgs; do
     if [ -n "${CI_OVERRIDE_CHANNEL:-}" ]; then
       info "Override channel set; retrieving ${pkg} from ${CI_OVERRIDE_CHANNEL}"
-      _hab install --channel="${CI_OVERRIDE_CHANNEL}" ${pkg} || {
+      _hab install --channel="${CI_OVERRIDE_CHANNEL}" "${pkg}" || {
         info "Package not found in ${CI_OVERRIDE_CHANNEL}; falling back to stable"
-        _hab install ${pkg}
+        _hab install "${pkg}"
       }
     else
-      _hab install $pkg
+      _hab install "$pkg"
     fi
   done
   
-  local bash_path=$(_pkgpath_for core/bash)
-  local coreutils_path=$(_pkgpath_for core/coreutils)
```

Comment:
We don't want these vars to be local any more?

Reply:
This file is POSIX sh (note the `#!/bin/sh`), which doesn't support `local`. See https://github.com/koalaman/shellcheck/wiki/SC2039

What is particularly dangerous is that POSIX sh silently ignores the `local` keyword (at least in my experience with the implementation we've used)